package.json

```diff
@@ -55,6 +57,8 @@
     "@types/node": "^22.13.5",
     "@types/uri-templates": "^0.1.34",
     "@types/yargs": "^17.0.33",
+    "@valibot/to-json-schema": "^1.0.0",
```

Comment:
adding all of these dependencies seems wrong.
They should be [optional peer dependencies](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta).

Reply:
These are dev dependencies for `src/examples/addition.ts`

Reply:
I agree @qcho we need to add @valibot/to-json-schema to optional peer deps.
And, I'd love to add zod/zod-to-json-schema to peer deps